src/plugins/charts/README.md

```diff
@@ -26,72 +26,4 @@ Truncated color mappings in `value`/`text` form
 
```

Comment:
It's irrelevant with this PR but I saw on L21 that there is a typo Funciton instead of Function and retrieve instead of retrieve. Do you want to also correct it in this PR ?

Reply:
Fixed in d82c6b8